exercises/G-conditionals-2/exercise-1.js

```diff
@@ -6,7 +6,13 @@
   - if number is more or equal to zero, return the word "positive"
 */
 
-function negativeOrPositive(number) {}
+function negativeOrPositive(number) {
+  if (num >= 0) {
```

Comment:
This isn't correct. Have you tried running this code? :)

Reply:
Typo fixed :-)